src/cluster_copier.cc

```diff
@@ -667,7 +667,7 @@ namespace grpc_labview {
 
                     for (int x = 0; x < count; ++x)
                     {
-                        auto data = (*array)->bytes<LVCluster*>(nestedMetadata->clusterSize * x);
+                        auto data = (LVCluster**)(*array)->bytes(nestedMetadata->clusterSize * x, nestedMetadata->alignmentRequirement);
```

Comment:
This is not `LVCluster**`. It is just `LVCluster*`. It doesn't change execution since all you do with it is cast it to `int8_t*` below, but it is wrong.